clowdapp.yaml

```diff
@@ -311,7 +311,7 @@ objects:
             "policies"."id",
             "profiles"."ref_id",
             "profiles"."name",
-            "accounts"."account_number",
+            "accounts"."org_id",
```

Comment:
This would not be as easy. At least we need to let know about this change to data engineers.

Reply:
This change is also already requested in https://github.com/RedHatInsights/compliance-backend/pull/1318.